apps/keyboard/js/layouts/emoji.js

```diff
@@ -0,0 +1,225 @@
+Keyboards.emoji = {
+  label: 'Emoji',
+  types: ['text', 'url', 'email'],
+  menuLabel: 'Emoji',
+  pages: [{   // Page 0 - People
+    emojiRange: [128512, 128591],
```

Comment:
This will also allow us not to tie our pages to Unicode ranges, which does not scale in the long term.